shared/src/event_handling.rs

```diff
@@ -170,11 +170,7 @@ where
     S: EventStoring<C::Event> + Send + Sync,
 {
     async fn run_maintenance(&self) -> Result<()> {
-        self.lock()
-            .await
-            .update_events()
-            .await
-            .context("event update error")
```

Comment:
This is the 🔑  change to avoid over generalizing the error message and losing track of what really happened.